gphoto2/Makefile.am

```diff
@@ -103,35 +106,22 @@ gphoto2_webapi_SOURCES      += range.c range.h
 gphoto2_webapi_SOURCES      += mongoose/mongoose.c mongoose/mongoose.h
 gphoto2_webapi_SOURCES      += server.c server.h
 
+gphoto2_webapi_CPPFLAGS     += -DPROGNAME=\"gphoto2-webapi\"
+gphoto2_webapi_CPPFLAGS     += -DGPHOTO2_WEBAPI
+
+# FIXME: Does interactive config make sense for gphoto2-webapi?
```

Comment:
@thorsten-l Does interactive config make sense for gphoto2-webapi?

Reply:
may be it would be nice to store the values for auth- (enabled,user,password) into $HOME/.gphoto/settings via config

Reply:
Passing passwords via command line on Linux makes them visible to anyone on the same machine who runs "ps", so a better way to set the password is necessary anyway.